src/config.json

```diff
@@ -734,6 +734,7 @@
     "cactus.chat"
   ],
   "blacklist": [
+    "xn--mtamsk-a.com",
```

Comment:
What URL was this meant to be? In trying to convert this back into unicode, I'm getting an error, saying that this is invalid punycode.

Reply:
And the build is failing because this isn't a valid domain